From b49a102074087e6680f330fcf61cebdeeac306b7 Mon Sep 17 00:00:00 2001
From: Ben Cooksley <bcooksley@kde.org>
Date: Mon, 7 Feb 2022 06:25:33 +1300
Subject: [PATCH] Purge all references to download.kde.org, replacing them with
 the correct endpoint of autoconfig.kde.org.

CCMAIL: plasma-devel@kde.org
CCMAIL: distributions@kde.org
(cherry picked from commit 1ab87dbf43bb51f21e2cd9e0188cb482a541734b)
---
 components/shellprivate/wallpaperplugin.knsrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/shellprivate/wallpaperplugin.knsrc b/components/shellprivate/wallpaperplugin.knsrc
index 1495b6e27..3ed59f75d 100644
--- a/components/shellprivate/wallpaperplugin.knsrc
+++ b/components/shellprivate/wallpaperplugin.knsrc
@@ -42,7 +42,7 @@ Name[x-test]=xxWallpaper Pluginsxx
 Name[zh_CN]=壁纸插件
 Name[zh_TW]=桌布外掛程式
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=Plasma Wallpaper Plugin
 StandardResource=tmp
 Uncompress=kpackage
-- 
GitLab

